controllers/fenceagentsremediation_controller_test.go

```diff
@@ -133,29 +133,37 @@ var _ = Describe("FAR Controller", func() {
 			It("should have finalizer and taint", func() {
 				farNamespacedName := client.ObjectKey{Name: node01, Namespace: defaultNamespace}
 				nodeNamespacedName := client.ObjectKey{Name: node01}
+				By("Searching for remediation taint")
 				Eventually(func() bool {
 					Expect(k8sClient.Get(context.Background(), nodeNamespacedName, node)).To(Succeed())
 					Expect(k8sClient.Get(context.Background(), farNamespacedName, underTestFAR)).To(Succeed())
 					res, _ := cliCommandsEquality(underTestFAR)
-					return controllerutil.ContainsFinalizer(underTestFAR, v1alpha1.FARFinalizer) && utils.TaintExists(node.Spec.Taints, &farNoExecuteTaint) && res
-				}, 1*time.Second, 500*time.Millisecond).Should(BeTrue(), "finalizer and taint should be added, and command format is correct")
+					return utils.TaintExists(node.Spec.Taints, &farNoExecuteTaint) && res
+				}, 20*time.Millisecond, 10*time.Millisecond).Should(BeTrue(), "taint should be added, and command format is correct")
```

Comment:
Nit: retry of every 10ms over a period of 20ms seems like cutting it really close, which in part defeating the use of `Eventually`, maybe use a longer period ? (first param)

Reply:
I will change to 100ms